### PR TITLE
bitbox-integration

### PR DESCRIPTION
**Pull request by 13Homer13**
I added bitbox guide to bitcoin integrations.

### CODE REVIEW

**Comment by 13Homer13** [comment]
I think I can add note that only connecting node via tor is available?

**Comment by gStart9** [comment]
I think if we're going to insert bitbox, we need a comprehensive guide -- so, ie our own guide.  It should be easy to adapt from theirs since it's already mostly done, but a few things like where to look in the StartOS Web UI for your electrum address need to be added in, otherwise this guide may cause more support rather than less.

**Comment by kn0wmad** [comment]
At some point, we'll eventually need to generalize our guides.  It's not going to remain possible to maintain guides for all possible third party software.  For now, it may warrant a quick "add electrs to this field, in this format, etc" - with a link to their guide.

File: site/source/service-guides/bitcoin/bitcoin-integrations.rst [hunk 1]
```diff
@@ -24,7 +24,16 @@ BitBoxApp
  
 **Instructions**
  
- Follow the `guide <https://shiftcrypto.support/help/en-us/14-privacy/29-how-to-connect-the-bitboxapp-to-my-own-full-node>`__.
+#. In the BitBoxApp sidebar, select Settings > Enable tor proxy.
+#. Enable the proxy and confirm the proxy address (127.0.0.1:9050).
+#. Then restart the BitBoxApp in order for the new settings to take effect.
```
Comment (kn0wmad) [inline site/source/service-guides/bitcoin/bitcoin-integrations.rst]:
```suggestion
#. Restart BitBoxApp in order for the new settings to take effect.
```

File: site/source/service-guides/bitcoin/bitcoin-integrations.rst [hunk 2]
```diff
@@ -24,7 +24,16 @@ BitBoxApp
...
+#. In the BitBoxApp sidebar, select Settings > Advanced settings > Connect your own full node
+#. In the field “Enter the endpoint”, paste Tor address and the port from Electrs properties. For example: gwdllz5g7vky2q4gr45zGuvopjzf33czreca3a3exosftx72ekppkuqd.onion:50001
```
Comment (kn0wmad) [inline site/source/service-guides/bitcoin/bitcoin-integrations.rst]:
```suggestion
#. In the field “Enter the endpoint”, paste the Tor address and port from your server's Electrs > Properties page. For example: gwdllz5g7vky2q4gr45zGuvopjzf33czreca3a3exosftx72ekppkuqd.onion:50001
```

File: site/source/service-guides/bitcoin/bitcoin-integrations.rst [hunk 3]
```diff
@@ -24,7 +24,16 @@ BitBoxApp
...
+#. Click “Check”, you should be prompted with the message “Successfully established a connection”.
```
Comment (kn0wmad) [inline site/source/service-guides/bitcoin/bitcoin-integrations.rst]:
```suggestion
#. Click “Check,” and you will be prompted with the message “Successfully established a connection.”
```

**Review by kn0wmad — CHANGES_REQUESTED** [review]

File: site/source/service-guides/bitcoin/bitcoin-integrations.rst [hunk 4]
```diff
@@ -24,7 +24,16 @@ BitBoxApp
...
+#. Click “Add” to add your node to the node list at the top of the page.
+#. Remove the other servers if you want to exclusively connect to your own node.
+
+Guide adopted from `Shiftcrypto <https://shiftcrypto.support/help/en-us/14-privacy/29-how-to-connect-the-bitboxapp-to-my-own-full-node>`__.
```
Comment (gStart9) [inline site/source/service-guides/bitcoin/bitcoin-integrations.rst]:
```suggestion
Guide adapted from `Shiftcrypto<https://shiftcrypto.support/help/en-us/14-privacy/29-how-to-connect-the-bitboxapp-to-my-own-full-node>`__.
```

**Review by kn0wmad — APPROVED** [review]

**Pull request merged**